src/main/java/emissary/output/filter/AbstractFilter.java

```diff
@@ -23,9 +23,6 @@
  * Provides the base mechanism for a drop off filter
  */
 public abstract class AbstractFilter implements IDropOffFilter {
-    /** A static convenience logger */
-    protected static Logger slogger = LoggerFactory.getLogger(AbstractFilter.class);
```

Comment:
These don't seem to have content, just types, etc. I might have to revisit what we say is content, but I'm not sure this class counts.